### ISSUE TITLE
Self signed certificates

### ISSUE THREAD

**Issue by mr-manuel** [rzrbld/adminio-api#20]
Hello,

would it be possible to add a switch to ignore SSL/TLS verification for MinIO server or a folder where the root CA can be placed, like in the MinIO docker containers?

If this is already possible, can you please add this in documentation?

Edit: I‘m using the latest docker container

**Comment by rzrbld** [comment]
Hi @mr-manuel by now there is no such functionality. In a proper way to achieve this, you need to add your self-signed ca to trusted certs in docker image. Just like in issue #17 .
It kind a looks like this:
```
...
RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
COPY ./mycert.crt /usr/local/share/ca-certificates/mycert.crt
RUN update-ca-certificates
...
```

**Comment by mr-manuel** [comment]
Ok, thank you. Would it be possible to add this for a future release?

**Comment by rzrbld** [comment]
ok, i will look how to implement this

**Comment by mr-manuel** [comment]
Thank you very much!

**Comment by BuJo** [comment]
I've implemented something like this in #25 and can probably be closed on the next release.

**Comment by rzrbld** [comment]
Thanks to @BuJo this functionality available in [v1.84](https://github.com/rzrbld/adminio-api/releases/tag/v1.84)
